src/main/java/com/crowdin/cli/commands/picocli/TaskAddSubcommand.java

```diff
@@ -0,0 +1,69 @@
+package com.crowdin.cli.commands.picocli;
+
+import com.crowdin.cli.client.ClientTask;
+import com.crowdin.cli.commands.Actions;
+import com.crowdin.cli.commands.NewAction;
+import com.crowdin.cli.properties.ProjectProperties;
+
+import org.apache.logging.log4j.util.Strings;
+import picocli.CommandLine;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@CommandLine.Command(
+        name = CommandNames.TASK_ADD
+)
+class TaskAddSubcommand extends ActCommandTask {
+
+    public static final String TRANSLATE_TASK_TYPE = "translate";
+    public static final String PROOFREAD_TASK_TYPE = "proofread";
+    @CommandLine.Parameters(descriptionKey = "crowdin.task.add.title")
+    private String title;
+
+    @CommandLine.Option(names = {"--type"}, paramLabel = "...", descriptionKey = "crowdin.task.add.type")
+    private String type;
+
+    @CommandLine.Option(names = {"--language"}, paramLabel = "...", descriptionKey = "crowdin.task.add.language")
+    private String language;
+
+    @CommandLine.Option(names = {"--file-id"}, descriptionKey = "crowdin.task.add.file-id", split = ",")
```

Comment:
`file-id` -> `file`

Use the `crowdin string add` as a reference

Reply:
done